### ISSUE TITLE
BERT: linear(): argument 'input' (position 1) must be Tensor, not NoneType

### ISSUE THREAD

**Issue by ajhepburn** [pytorch/captum#696]
## 🐛 Bug

I've been following your BERT tutorial and managed to get LayerConductance working previously, and had previously passed in the tokens instead of the input embeddings before the tutorial was updated (due to the HuggingFace >4.0 break). I've upgraded to 4.8.1 recently and changed my code accordingly so that I'm passing only the input_embeddings and refs. However, I get a very strange result when I pass in my trained classifier.

I get the error `linear(): argument 'input' (position 1) must be Tensor, not NoneType`.

The Pytorch module I pass in is a simple classifier, wrapped with a dropout and linear output layer as follows:

```
class TransformerClassifier(nn.Module):

    def __init__(self, base):
        super(TransformerClassifier, self).__init__()
        self.base = base
        self.drop = nn.Dropout(p=0.3)
        self.out = nn.Linear(self.base.config.hidden_size,
                                      self.base.config.num_labels)

    @classmethod
    def create(cls, model_name, model_type, model_config):
        model = model_type.from_pretrained(
            model_name,
            config=model_config,
        )
        return cls(base=model)

    def forward(self, **kw):
        x = self.base(**kw)
        x = x.pooler_output
        x = self.drop(x)
        return self.out(x)
```

I've tried to debug it to see if it's maybe something I've done myself but it doesn't seem to exit the HuggingFace transformers stage so it's not my linear output layer, I'm wondering if there's something input related I'm not passing in.

## To Reproduce

1. Follow BERT SQuAD tutorial (with slight modifications to remove position information for classification)

```
def squad_pos_forward_func2(input_emb, attention_mask=None):
    pred = captum_model.model(inputs_embeds=input_emb, attention_mask=attention_mask)
    return pred.max(1).values

for i in range(model.base.config.num_hidden_layers):
    lc = LayerConductance(squad_pos_forward_func2, model.base.encoder.layer[i])
    layer_attributions_start = lc.attribute(inputs=input_embeddings, baselines=ref_input_embeddings, additional_forward_args=(attention_mask))
    break
```

## Environment
Describe the environment used for Captum

```

 - Captum / PyTorch Version (e.g., 1.0 / 0.4.0): 1.8.1
 - OS (e.g., Linux): Using Jupyter notebook in external Kubernetes cluster
 - How you installed Captum / PyTorch (`conda`, `pip`, source): conda
 - Build command you used (if compiling from source): Dockerfile?
 - Python version: 3.8
 - CUDA/cuDNN version: 7.6.5
 - GPU models and configuration: Nvidia
 - Any other relevant information:

I tried to fill out the template as best as I could so apologies if there's much let out, let me know and I'll provide anything additional.

**Comment by ajhepburn** [comment]
Never mind, I am very careless and still had the `configure_interpretable_embedding_layer` from the older version of the tutorial, removed this step and all is fine!
